pkg/apis/alerting/v1alpha/alerting.proto

```diff
@@ -203,7 +221,15 @@ message AlertCondition {
     AlertConditionSystem system = 5;
   }
   optional string notificationId = 6;
-  EndpointImplementation details = 7; 
+  EndpointImplementation details = 7;
+  // Set & Unset via the silence API
+  optional SilenceInfo silence = 8;
```

Comment:
The `optional` keyword is not used in proto3

Reply:
Removed